app/controllers/authors_controller.rb

```diff
@@ -0,0 +1,22 @@
+class AuthorsController < ActionController::Base
+  def new
+    @author = Author.new
+  end
+
+  def create
+    l_params = {
+      first_name: params[:first_name],
+      last_name: params[:last_name],
+      email: params[:email]
+    }
+
+    @author = Author.new(l_params)
+
+    if @author.valid?
+      RequestJob.perform_later l_params if @author.valid?
```

Comment:
duplicate condition

Reply:
Try completely removing this line. Your existing tests won't catch it.

Reply:
Makes sense. I thought testing requests would be enough for this project.

Reply:
My bad. Forgot to delete that one :)

Reply:
Well, at least `request specs` will catch it because I am expecting corresponding status codes returned for valid and not valid params. But I admit my fault for not testing the controller itself.

Reply:
> duplicate condition

My bad. Forgot to delete that one :)

Reply:
> Try completely removing this line. Your existing tests won't catch it.

Well, at least request specs will catch it because I am expecting corresponding status codes returned for valid and not valid params. But I admit my fault for not testing the controller itself.